docs/src/main/sphinx/release/release-390.md

```diff
@@ -0,0 +1,44 @@
+# Release 390 (13 July 2022)
+
+## General
+
+* Add support for setting view comments via the `COMMENT ON VIEW` statement. ({issue}`8349`)
+* Improve performance of queries with the `UNNEST` clause. ({issue}`10506`)
+* Fix potential query failure when the `force-spilling-join-operator`
+  configuration property or `force_spilling_join` session property are enabled. ({issue}`13123`)
+
+## Cassandra connector
+
+* Add mapping for Cassandra `inet` type to Trino `ipaddress` type. ({issue}`851`)
+* Remove unused `cassandra.load-policy.use-token-aware`,
+  `cassandra.load-policy.shuffle-replicas`, and
+  `cassandra.load-policy.allowed-addresses` configuration properties. ({issue}`12223`)
+
+## Delta Lake connector
+
+* Expose AWS Glue metastore statistics via JMX. ({issue}`13087`)
+* Fix query failure when pushing `IS NULL` and `IS NOT NULL` predicates onto
+  numeric types in Glue. ({issue}`13124`)
+
+## Hive connector
+
+* Expose AWS Glue metastore statistics via JMX. ({issue}`13087`)
+* Fix query failure when pushing `IS NULL` and `IS NOT NULL` predicates onto
+  numeric types in Glue. ({issue}`13124`)
+* Fix and re-enable S3 Select for uncompressed files. ({issue}`12633`)
```

Comment:
Maybe use Amazon S3 Select  


also .. 

Fix and re-enable usage of Amazon S3 Select